### DISCUSSION TITLE
How to get off of OnlyOffice

### DISCUSSION THREAD

**Discussion by jgb1128** [CC-in-the-Cloud/Admin#85] [Q&A]
Team:

I really want to fully migrate to GitHub, leaving it only as an archive.  This discussion feature in GitHub replicates OnlyOffice's discussion feature. The ONLY advantages I can see to OnlyOffice vs GitHub is the easy to read "membership" list, and the notion of some kind of access control.  Just to test this, I'm posting this exact same announcement in Git and I'd ask Git users to just acknowledge if they were notified of this or only went to check it out after I announced it here.

Thanks,

--Josh

**Comment by bharveyTX** [comment]
FYI, anyone can change their notification settings for this repo by using the "Watch" feature in the upper-right-hand corner of the UI. See the screenshot below:

![Screenshot 2023-02-23 at 6 00 33 PM](https://user-images.githubusercontent.com/64159025/221059292-ea48c6d4-3526-4a23-959b-4e8efad219ff.png)

**Reply by fahmed11** [reply]
Well, I hopped on here following the OnlyOffice link. I didn't get the Git notification, because  only the first option is checkmarked in my Watch setting.

**Reply by kr15tyk** [reply]
Same for me. I setup a custom "watch" for future notifications.

**Reply by GY5318** [reply]
I did not get either and checked my Junk mail too.

**Reply by heimannrj** [reply]
My notification settings were also not set appropriately.  Changed to "All Activity"

**Reply by jfmyerrors** [reply]
I had a similar experience to Farid.  Came here after read the announcement on OnlyOffice..  I had not received a GitHub notification.  So I changed my  Watch setting from the first option to the second.  I will worry about customizing later.

**Reply by jfmyerrors** [reply]
Another possibility could be that some of us had simply forgotten to also select "Follow" at the top level.  When I updated my "Watch" settings, I also changed to Follow.   King Ables said that he had changed both as well.  So neither of two of us can say which change fixed the notifications.

**Comment by kingoatsec** [comment]
I only got a notification from the posting on OnlyOffice. I came here to find I had the same setting in my watch list as Brandon showed, when participating or mentioned. i've changed it to All Activity.

**Reply by mmbarry** [reply]
I experienced the same as King and now should be notified of all activity going forward.

**Comment by GallagherTom** [comment]
I saw the message through a GitHub notification before I saw it in OnlyOffice.
